gradle.properties

```diff
@@ -25,7 +25,7 @@ android.enableJetifier=true
 android.useAndroidX=true
 org.gradle.jvmargs=-Xmx1536m
 # Required by Robolectric.
-android.enableUnitTestBinaryResources=true
+#android.enableUnitTestBinaryResources=true
```

Comment:
QQ: Do the tests still run fine given this change? I feel like I needed this at one point?

Reply:
according to [docs](http://robolectric.org/getting-started/), this flag is no longer needed